q2_feature_table/plugin_setup.py

```diff
@@ -18,7 +18,14 @@
     name='feature-table',
     version=q2_feature_table.__version__,
     website='https://github.com/qiime2/q2-feature-table',
-    package='q2_feature_table'
+    package='q2_feature_table',
+    citation_text=('The Biological Observation Matrix (BIOM) format or: how '
```

Comment:
Right, but this uses code developed for that manuscript? I added this as I had support for it from @gregcaporaso but I'm also okay moving the citation elsewhere if that's the consensus